From 6b979dbe842abd40c7bc8b2c7355f7859bb414cf Mon Sep 17 00:00:00 2001
From: Ryan Pavlik <rpavlik@iastate.edu>
Date: Fri, 18 Jan 2013 15:36:40 -0600
Subject: [PATCH 1/2] Ignore IWYU directory

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 6c34e37..56d91a0 100644
--- a/.gitignore
+++ b/.gitignore
@@ -30,6 +30,7 @@ cscope.out
 #==============================================================================#
 # Clang extra user tools, which is tracked independently (clang-tools-extra).
 tools/extra
+tools/include-what-you-use
 # Sphinx build products
 docs/_build
 docs/analyzer/_build
-- 
1.8.2

