From 852e71ce91f8b64b725519c97cc978a341fe9081 Mon Sep 17 00:00:00 2001
From: Ryan Pavlik <rpavlik@iastate.edu>
Date: Thu, 28 Mar 2013 12:56:01 -0500
Subject: [PATCH 2/2] Compile IWYU as well.

---
 tools/CMakeLists.txt |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index eb5e366..57e7940 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -7,6 +7,8 @@ add_subdirectory(driver)
 add_subdirectory(clang-check)
 add_subdirectory(clang-format)
 
+add_subdirectory(include-what-you-use)
+
 # We support checking out the clang-tools-extra repository into the 'extra'
 # subdirectory. It contains tools developed as part of the Clang/LLVM project
 # on top of the Clang tooling platform. We keep them in a separate repository
-- 
1.7.1

